### ISSUE TITLE
Unable to resolve module

### ISSUE THREAD

**Issue by dnish** [Urigo/meteor-client-bundler#10]
Hey,
within my React Native app, I get the following error message:

```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1579): UnableToResolveError: Unable to resolve module `./install-packages.js` from `/Users/me/Desktop/Entwicklung/app/app/node_modules/meteor-client.js`
```

Did I forget something?

**Comment by DAB0mB** [comment]
@dnish can you please tell in a bit more detail what actions did you take? Did you bundle? How? Can you please show me your entry script and how you loaded meteor-client? What operation system do you use and what version of meteor do you run?

**Comment by dnish** [comment]
Hey,
yeah, the bundle is saved within <code>node_modules/meteor_client.js</code>. Before starting the React Native app, I did:

meteor-client bundle --config mcb.config.json

Then I've added the following within index.android.js (at the top):

    import "meteor-client";


After that, I've run:

    react-native start

and at 99,7% I get the error above. I've already tried to delete the node_modules folder and to clear watchmen. I'm using OSX 10.12.3 - Meteor 1.4.3.2

**Comment by DAB0mB** [comment]
OK, can you please show me the config file mcb.config.json?

**Comment by dnish** [comment]
Sure:
```
{
  "runtime": {
"DDP_DEFAULT_CONNECTION_URL": "http://192.168.178.28:3000"
},
"import": [
"meteor-base@1.0.4",
"mongo@1.1.14",
"reactive-var@1.0.11",
"jquery@1.11.10",
"tracker@1.1.1",
"standard-minifier-css@1.3.2",
"standard-minifier-js@1.2.1",
"es5-shim@4.6.15",
"ecmascript@0.6.1",
"shell-server@0.2.1"
]
}
```

Server is also running on dev machine. I've already tried it without config and linking it to the Meteor directory, same error message.

**Comment by DAB0mB** [comment]
And if you will provide me with the log produced by MCB to the stdout, it would help as well

**Comment by dnish** [comment]
I only get a stdout log if I run the meteor-client bundle command, so the output is:

```
Denniss-iMac:app dennis$ meteor-client bundle --config meteor-client.config.json
Created a new Meteor app in '/var/folders/vs/q1_brjt151z2p_kmdjslvp9h0000gn/T/tmp-4631YydS9UDYP247'.

To run your new app:                          
  cd /var/folders/vs/q1_brjt151z2p_kmdjslvp9h0000gn/T/tmp-4631YydS9UDYP247
  meteor                                      
                                              
If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/learn                
                                              
meteor create --bare to create an empty app.  
meteor create --full to create a scaffolded app.
                                              
                                              
Changes to your project's package version selections:
                                              
autopublish            removed from your project
blaze-html-templates   removed from your project
caching-compiler       removed from your project
caching-html-compiler  removed from your project
fastclick              removed from your project
insecure               removed from your project
launch-screen          removed from your project
mobile-experience      removed from your project
mobile-status-bar      removed from your project
templating             removed from your project
templating-compiler    removed from your project
templating-runtime     removed from your project
templating-tools       removed from your project

                                              
WARNING: The output directory is under your source tree.
         Your generated files may get interpreted as source code!
         Consider building into a different directory instead
         meteor build ../output
                                              
```

**Comment by DAB0mB** [comment]
@dnish Good news. I fixed this issue, as seen on the following branch: https://github.com/Urigo/meteor-client-bundler/tree/feat/escape-commonjs. I've encountered a new problem with React-Native described in the following issue: https://github.com/Urigo/meteor-client-bundler/issues/12, therefore, I will close this issue, but know that I'm working on it :-) Hopefully you will be able to run React-Native with Meteor soon.
